go/apps/ctrl/services/build/backend/depot/create_build.go

```diff
@@ -295,28 +300,33 @@ func (s *Depot) getOrCreateDepotProject(ctx context.Context, unkeyProjectID stri
 	return depotProjectID, nil
 }
 
-func (s *Depot) processBuildStatus(
+func (s *Depot) processBuildStatusLogs(
```

Comment:
@coderabbitai create an issue for us to set a hard cap on how many logs we accept. Someone might try a denial of service by building an image with billions of rows of logs